tools/modules/software/module_wireguard.sh

```
@@ -66,16 +100,148 @@ function module_wireguard () {
+					echo "ip route add $subnet via \"\$CONTAINER_IP\"" >> /usr/local/bin/add-vpn-lan-routes.sh
+				done
+
+				cat > "/usr/local/bin/remove-vpn-lan-routes.sh" <<- EOT
+				#!/bin/bash
+				docker exec wireguard iptables -t nat -C POSTROUTING -o client -j MASQUERADE 2>/dev/null && \
+				docker exec wireguard iptables -t nat -D POSTROUTING -o client -j MASQUERADE || true
+				EOT
+				# Loop through each subnet and append a route command
+				IFS=',' read -ra SUBNETS <<< "$LOCAL_SUBNETS"
+				for subnet in "${SUBNETS[@]}"; do
+					echo "ip route show "$subnet" | grep -q "$subnet" && ip route del "$subnet" || true" >> /usr/local/bin/remove-vpn-lan-routes.sh
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)